pma_api/bin/heroku/7.22.2/MacOS/heroku

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
What is the purpose of this file? I haven't seen this before. Is it putting the CLI on Heroku?

Reply:
@jkpr This is for utilization of Heroku's backup/restore feature. It is not needed except for unit tests which utilize it.